dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -268,25 +238,24 @@ public void Test_Create_app_descriptor_Then_Create_App_Integration_Then_Delete_T
      * Expected Result: Pagination shows all available sites with no configurations and the default values from the yml are loaded instead.
      */
     @Test
-    public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Delete_One_Single_Secret() {
-
-        final SortedMap<String, ParamDescriptor> paramMap = ImmutableSortedMap.of(
-                "param1", ParamDescriptor
-                        .newParam("default", false, Type.STRING, "label", "hint", true),
-                "param2", ParamDescriptor
-                        .newParam("default", false, Type.STRING, "label", "hint", true),
-                "param3", ParamDescriptor
-                        .newParam("default", false, Type.STRING, "label", "hint", true)
-        );
+    public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Delete_One_Single_Secret()
+            throws IOException {
+
+        final AppDescriptorDataGen dataGen = new AppDescriptorDataGen()
+                .stringParam("param1", false,  true)
+                .stringParam("param2", false,  true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "param2" appears 9 times in this file; the first occurrence is on line 254](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)